controllers/egressfirewall.go

```diff
@@ -0,0 +1,158 @@
+package controllers
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	openshiftv1 "github.com/openshift/api/network/v1"
+	ovnv1 "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1"
+	"github.com/red-hat-storage/managed-fusion-agent/templates"
+	"github.com/red-hat-storage/managed-fusion-agent/utils"
+	corev1 "k8s.io/api/core/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func (r *ManagedFusionReconciler) reconcileEgressFirewall() error {
+	if !r.AvailableCRDs[EgressFirewallCRD] {
+		r.Log.Info("EgressFirewall CRD not present, skipping reconcile for egress firewall")
+		// return nil
```

Comment:
If this is just a comment can we remove it? If not can we add it back(look for other places as well)?